packages/material-ui/src/styles/withStyles.js

```diff
@@ -48,7 +48,11 @@ function getDefaultTheme() {
     return defaultTheme;
   }
 
-  defaultTheme = createMuiTheme();
+  defaultTheme = createMuiTheme({
+    typography: {
+      suppressWarning: true,
```

Comment:
@oliviertassinari 
I just found this. Why was this snuck in in this PR? We had this discussion over and over. If you don't care about deprecation warnings just say it and we can leave this discussion behind. I'm tired of watching every commit so that you don't work against the determined deprecation behavior in the RFC.

[no more deprecation warnings](https://codesandbox.io/s/k5qr211785)

Reply:
The motivation is the same of the fix proposed in https://github.com/mui-org/material-ui/issues/13657. Let me know what you think about it.

Reply:
Why was it introduced in this pull request? Hum, probably because I want to minimize the numbers of commits I'm doing, I try to batch them so our contribution graph looks more balanced.

Reply:
Oh yes, that was it. I was probably running the benchmark test suite, without providing a custom them then I saw this warning pop. I think that it's important people can use Material-UI without having to provide a theme.

Reply:
Do you want to change the tradeoff?